english/movie-ripoffs/hogwarts/hogwarts.md

```diff
@@ -25,4 +25,8 @@ from other students as the elderly wizard leaps up the staircase in misguided pu
 
 "Don't forget to put on your invisible cloak, Sir" calls Caroline as he disappears out of sight.  
 
-"We just tell him that he's invisible when he's wearing it - he's soooo gullible." giggles Caroline, rolling her eyes.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.